modules/ROOT/pages/profile-creation.adoc

```diff
@@ -82,10 +82,7 @@ You can duplicate a profile name in different files, but remember that you can o
 In the mapping section you can map Users / Groups / Roles / Memberships to your profile. You have to map your profile with elements of the active organization. +
 ⚠  Nodes in the group tree are *independent* ⚠  It means that mapping `/acme` to your profile won't map `/acme/hr` to it. Each node has to be mapped individually to be taken into account. However, you can easily add a whole part of your organization, using the _add subtree_ button:  select a node in the tree, and then click on _add subtree_.
 
-The portal menu entries can be edited for custom profiles. +
-Pages proposed are those provided by default in the Bonita Runtime and those created with the UI Designer. You can reference a custom page only if it is imported in the Bonita Runtime, and you will have to write its exact identifier.
-
-The Applications section provides a quick access to the applications mapped to this profile. +
+The Applications section provides quick access to the applications mapped to this profile. +
 To map a new application to this profile, you have to open the application and then choose this profile in the profile field of the xref:applicationCreation.adoc[application descriptor editor]
```

Comment:
profile, too much profile. 

To map a new application to this profile, you have to open the application and then choose this profile in the profile field of the xref:applicationCreation.adoc[application descriptor editor] => 
For adding an application to a profile, open the application and assign the required profile by using xref:applicationCreation.adoc[application descriptor editor].